gunicorn/util.py

```diff
@@ -363,7 +363,7 @@ def import_app(module):
     except NameError:
         if is_debug:
             traceback.print_exception(*sys.exc_info())
-        raise AppImportError("Failed to find application: %r" % module)
+        raise AppImportError("Failed to find application object %r in %r" % (obj,module) )
```

Comment:
Style nits: Please add a space after `,` and remove the space between the two closing parens.

Reply:
OK